src/RoomsSDKAdapter.js

```diff
@@ -134,4 +144,119 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
 
     return this.getRoomObservables[ID];
   }
+
+  /**
+   * Invoke request to listActivities on adapter
+   *
+   * @param {object} Params to send to conversation api method
+   * @returns
+   * @private
+   */
+  fetchActivities({roomId, earliestActivityDate}) {
+    const {limit} = this;
+    const conversationId = deconstructHydraId(roomId).id;
+
+    logger.debug('ROOM', 'fetchActivities()', ['called with', {conversationId, limit, earliestActivityDate}]);
+
+    let maxDate = earliestActivityDate;
+
+    if (maxDate === null) {
+      maxDate = undefined;
+    }
+
+    return this.datasource.internal.conversation.listActivities({
+      conversationId,
+      limit,
+      lastActivityFirst: true,
+      maxDate,
+    });
+  }
+
+  /**
+   * Handles checking if there is more activities to fetch.
+   *
+   * @param {string} roomId The room id to check
+   * @returns {boolean} True if there is more activities
+   */
+  hasMoreActivities(roomId) {
```

Comment:
There is also this style pattern in the repo to put all public methods at the end and all privates before public methods.